content/en/docs/gke/deploy/reasons.md

```diff
@@ -7,10 +7,9 @@ weight = 10
 Running Kubeflow on GCP brings you the following features:
 
   * You use
-    [Deployment Manager](https://cloud.google.com/deployment-manager/docs/) to
+    [Cloud Native Resource Manager](https://cloud.google.com/config-connector/docs) to
```

Comment:
Suggestion:

```diff
- * You use
+ * The [Cloud Native Resource Manager](https://cloud.google.com/config-connector/docs) enables you to
declaratively manage all non-Kubernetes resources (including the GKE
cluster)
```

(Notice there's no full stop, for consistency)

Also, these sounds like benefits, not just features, so maybe amend line 7 to:

```markdown
Running Kubeflow on GCP has the following benefits:
```

Reply:
I didn't understand your comment about the period. It looks like all the bullets have periods. So should there be one at the end if you want consistency?

Reply:
Ah yes, if all of them have periods, then no worries. It was a long day!